README.md

```diff
@@ -1 +1,79 @@
-# portfolio_mobile_day3
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] [REQUIRED] Please write your project name(e.g: `Portfolio`) instead of `"Project Name"`😃